phalcon/cache/backend/redis.zep

```diff
@@ -88,13 +99,21 @@ class Redis extends Backend implements BackendInterface
 		}
 
 		if !isset options["statsKey"] {
-			// Disable tracking of cached keys per default
-			let options["statsKey"] = "";
+			let options["statsKey"] = "_PHCR";
```

Comment:
Because it is consistent with the document. Current implementation is not consistent with other cache backends. I will modify the document instead.